src/scaffoldmaker/utils/read_vagus_data.py

```diff
@@ -290,9 +290,11 @@ def load_vagus_data(region):
     :param region: Zinc region for model definition.
     return: Provided the input file is supplied, it returns a data region with input data.
     """
+
+    logger = logging.getLogger(__name__)
     data_region = region.getParent().findChildByName('data')
     if not data_region.isValid():
-        print("Invalid input data file", file=sys.stderr)
+        logger.warning("Missing input data.")
```

Comment:
Minor note: this warning will be followed by errors which assume the data was present.
I'm not fussing now as we hope to eventually have built-in default data.